app/views/foods/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<!-- app/views/foods/index.html.erb -->
+
+<%= stylesheet_link_tag "foods", "data-turbo-track": "reload" %>
+<h2>Food List</h2>
```

Comment:
[OPTIONAL] It is a good practice to ensure that each page has its own `h1` tag. But there isn't any here